### DISCUSSION TITLE
Consider creating a Distributed System Analysis "round table" repository for governance

### DISCUSSION THREAD

**Discussion by portante** [distributed-system-analysis/pbench#3061] [Polls]
The [Arcalot](https://github.com/arcalot) organization maintains their governance, charters, etc. in a repository called the [Arcalot Round Table](https://github.com/arcalot/arcalot-round-table).

As a chair person on their board, I have participated in their process a bit and it is very compelling to me, enough so that I'd like to establish a similar governance for this organization, Distributed System Analysis, where the participation in the code repositories within this organization can be properly governed.

Please review the [Arcalot Round Table](https://github.com/arcalot/arcalot-round-table) and how it might be adapted and applied to this organization governing the Pbench repository.

**Comment by dbutenhof** [comment]
I'm not entirely thrilled about arcaflow's implementation, with the directory of all proposals and apparently a summary file showing status. This makes it a bit difficult to easily determine the status of proposals. I'd prefer to have a clean separation between "proposals" and "accepted decisions".

However, in theory, the idea sounds almost essential as we gain external traction.
